### ISSUE TITLE
websocket server can't send correct message

### ISSUE THREAD

**Issue by Ulyssesyi** [swoole/swoole-src#2933]
Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a simple script for reproducing the error.
install swoole by pecl and swoole version is 4.4.10. Below is my code.
```
<?php
$server = new Swoole\WebSocket\Server("0.0.0.0", 9051);

$server->on('open', function (Swoole\WebSocket\Server $server, $request) {
    echo "server: handshake success with fd{$request->fd}\n";
});

$server->on('message', function (Swoole\WebSocket\Server $server, $frame) {
    echo "receive from {$frame->fd}:{$frame->data},opcode:{$frame->opcode},fin:{$frame->finish}\n";
    $server->push($frame->fd, "this is server");
});

$server->on('close', function ($ser, $fd) {
    echo "client {$fd} closed\n";
});

$server->start();
```
```
<!DOCTYPE html>
<html>
<head>
	<title>ws</title>
</head>
<body>

</body>
<script type="text/javascript">
	var ws = new WebSocket("ws://127.0.0.1:9051");

	ws.onopen = function(evt) { 
	  console.log("Connection open ..."); 
	  ws.send("Hello WebSockets!");
	};

	ws.onmessage = function(evt) {
	  console.log( "Received Message: " + evt.data);
	  ws.close();
	};

	ws.onclose = function(evt) {
	  console.log("Connection closed.");
	};      
</script>
</html>
```

2. What did you expect to see?
web can recevie the message from server.

3. What did you see instead?
console output Connection open ... and then throw an error 
WebSocket connection to 'ws://127.0.0.1:9051/' failed: Could not decode a text frame as UTF-8.

4. What version of Swoole are you using (show your `php --ri swoole`)?
4.4.10

5. What is your machine environment used (including version of kernel & php & gcc) ?
centos 7/php 7.2.24/gcc 4.8.5

**Comment by newartix** [comment]
Same problem on Windows Docker VM, PHP 7.3.8, Swoole 4.4.9

**Comment by twose** [comment]
Fixed in v4.4.11
We will release a new version soon
